Maven.gitignore

```diff
@@ -15,3 +15,6 @@ buildNumber.properties
 .project
 # JDT-specific (Eclipse Java Development Tools)
 .classpath
+
+#This file records project specific settings vs. workspace preferences.
```

Comment:
Missing space `# This ...`.

Reply:
I used other sources of clarification, but I observed that the discussions you mentioned are mature and clear.